app/assets/javascripts/standalone/help.js

```diff
@@ -1,24 +1,23 @@
 // Hides and shows each page of Help
 
-function open_window(id) {
-   var element = document.getElementById(id);
+function openWindow(id) {
+   const element = document.getElementById(id);
```

Comment:
Use ES6 syntax.

Reply:
Yes, the progress tab JS is 100% ES6, for example and I confirmed previously with @vinaysshenoy that the minimum API version that our app supports has ES6 support.

Reply:
Perfect 👌